mrack.spec

```diff
@@ -0,0 +1,61 @@
+# Created by pyp2rpm-3.3.5
+%global srcname mrack
+
+Name:           %{srcname}
+Version:        0.6.0
+Release:        1%{?dist}
+Summary:        Multicloud use-case based multihost async provisioner for CIs and testing during development
+
+License:        Apache License 2.0
+URL:            https://github.com/neoave/mrack
+Source0:        https://github.com/neoave/mrack/releases/download/v%{version}/mrack-%{version}.tar.gz
+BuildArch:      noarch
+
+### Patches ###
+Patch0001:  0001-chore-set-boto-minimum-versions.patch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3dist(asyncopenstackclient) >= 0.8.1
+BuildRequires:  python3dist(beaker-client) >= 28
+BuildRequires:  python3dist(boto3) >= 1.11.17
+BuildRequires:  python3dist(botocore) >= 1.14.17
+BuildRequires:  python3dist(click) >= 7
+BuildRequires:  python3dist(pyyaml) >= 5
+BuildRequires:  python3dist(setuptools)
+
+%description
+mrack is a provisioning library for CI and local multi-host testing
+supporting multiple provisioning providers (e.g. OpenStack, libvirt, containers,
```

Comment:
we should probably not mention the stuff which is not-yet implemented (libvirt, containers)